apps/sage_intacct/queue.py

```
@@ -1,52 +1,41 @@
 import logging
 from datetime import datetime, timedelta, timezone
+from typing import List
 
 from django.db.models import Q
 from django_q.models import Schedule
 from django_q.tasks import Chain
 
 from apps.tasks.models import TaskLog, Error
-from apps.fyle.models import ExpenseGroup, Expense
+from apps.fyle.models import ExpenseGroup
 from apps.workspaces.models import Configuration, FyleCredential
```

Ruff (0.8.2)

`apps.workspaces.models.FyleCredential` imported but unused

Remove unused import: `apps.workspaces.models.FyleCredential`

(F401)